### DISCUSSION TITLE
Connecting using RSA_IDs

### DISCUSSION THREAD

**Discussion by scath999** [winfsp/winfsp#418] [Q&A] [answered]
Hi there,

Having trouble mounting a drive over ssh using an rsa_id.

I tried specifying the resource location as <account_name>@<my_domain>

At first winfsp asked me for a password (which I have), but entering the pass didn't work because I don't allow sign-ins with passwords on my server.

Now, it can't find the droplet at all.

Thoughts?

Cheers!

**Comment by billziss-gh** [comment]
This discussion list is for WinFsp and not SSHFS-Win.

Please see the instructions for `\\sshfs.k` [here](https://github.com/winfsp/sshfs-win#unc-syntax).

**Reply by scath999** [reply]
Apologies for posting in the wrong place - I haven't used these tools before, so I'm unsure exactly which part of the solution is managing what.

Thanks for pointing me in that direction - am asking there. Are you saying I don't need WinFSP? 

If so, no issue.

Thanks for replying.

**Comment by scath999** [comment, marked as answer]
Solved this using SiriLaki.

Cheers!
